### PR TITLE
updates to nextstrain tasks and workflows

### PR DESCRIPTION
**Pull request by dpark01**
- `augur export` task: adds optional `--lat-longs` and `--colors` tsv inputs and traits input
- adds new WDL task for `augur traits` and inserts it into the workflow
- adds new WDL task for `augur filter` -- not currently part of the main workflow
- fix up some parameter handling for augur align, augur tree, augur refine
- adds `meta` sections to all tasks, with links to original documentation about parameters

### CODE REVIEW

File: pipes/WDL/tasks/tasks_nextstrain.wdl [hunk 1]
```diff
@@ -307,6 +309,8 @@ task export_auspice_json {
             --metadata ~{metadata} \
             $NODE_DATA_FLAG ~{sep=' ' select_all([branch_lengths,nt_muts,aa_muts])}\
             --auspice-config ~{auspice_config} \
+            ${"--lat-longs" + lat_longs_tsv} \
```
Comment (tomkinsc) [inline pipes/WDL/tasks/tasks_nextstrain.wdl:312]: Should we be consistent about `~{}` vs `${}`, at least per-file? The [WDL spec](https://github.com/openwdl/wdl/blob/master/versions/1.0/SPEC.md#expression-placeholders) says `~{}` is preferred now (though I still like`${}`).

Comment (dpark01) [reply pipes/WDL/tasks/tasks_nextstrain.wdl]: true -- I'm new to this notation, and also didn't realize it applied to these other situations (with prefixes before the substitutions, or even the `sep`) but we should stay consistent at least witihin the task level

File: pipes/WDL/tasks/tasks_nextstrain.wdl [hunk 2]
```diff
@@ -85,9 +147,9 @@ task augur_mafft_align {
             --reference-sequence ~{ref_fasta} \
             --output ~{basename}_aligned.fasta \
             ~{true="--fill-gaps" false="" fill_gaps} \
-            ~{true="--existing-alignment " false="" existing_alignment} \
+            ~{"--existing-alignment " + existing_alignment} \
             ~{true="--remove-reference" false="" remove_reference} \
-            ~{true="--debug" false="" debug} \
+            --debug \
             --nthreads auto
         cat /sys/fs/cgroup/memory/memory.max_usage_in_bytes | tee MAX_RAM
```
Comment (schaluva) [inline pipes/WDL/tasks/tasks_nextstrain.wdl:154]: While testing the WDL the "tee MAX_RAM" had some errors so I wrote to a file and caught the output in a "File" in the output section.  
cat /sys/fs/cgroup/memory/memory.max_usage_in_bytes > MAX_RAM
  
    output {
        ...
        ...
       	File  max_ram_usage_in_bytes = "MAX_RAM"
For reference here is the error from Terra with the original line of code:
Screen Shot 2020-05-07 at 6 51 15 PM

Comment (dpark01) [reply pipes/WDL/tasks/tasks_nextstrain.wdl:154]: weird -- I wonder if the number is too big? Ok making the change, though I'm also separately dabbling with a more thorough heartbeat reporting mechanism.

**Pull request merged**